website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -96,12 +96,11 @@ That means that patch version numbers will likely differ between `dbt-core` and
 For example, you may find you're using `dbt-core==1.2.3` with `dbt-snowflake==1.2.0`. It is critical that you're using the latest patch available for both core and the adapter (v1.2.x). Use the `dbt --version` command to see which versions you have installed:
 ```
 $ dbt --version
-installed version: 1.2.3
-   latest version: 1.2.3
-
-Up to date!
+Core:
+  - installed: 1.5.3
+  - latest:    1.5.3 - Up to date!
 
 Plugins:
-  - snowflake: 1.2.0 - Up to date!
+  - snowflake: 1.5.2 - Up to date!
```

Comment:
Could we go all the way up to 1.6.0?

Reply:
sure thing! i can update the `Core` section to 1.6.0